.github/workflows/per-pr.yml

```diff
@@ -66,3 +66,61 @@ jobs:
 
       - name: Kurtosis Lint
         run: kurtosis lint ${{ github.workspace }}
+
+  assertoor:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Setup Kurtosis
+        run: |
+          echo "deb [trusted=yes] https://apt.fury.io/kurtosis-tech/ /" | sudo tee /etc/apt/sources.list.d/kurtosis.list
+          sudo apt update
+          sudo apt install kurtosis-cli
+          kurtosis analytics disable
+
+      - name: Run Starlark
+        run: kurtosis run ${{ github.workspace }} --enclave assertoor-${{ github.run_id }}-${{ inputs.id }} --args-file ./.github/tests/mix-assert.yaml --image-download always --non-blocking-tasks --verbosity DETAILED
```

Comment:
remove `-${{ inputs.id }}` from the enclave name
that input is probably not defined in this workflow.
The run_id should be enough to make the enclave name unique here